### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Cano-J**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,6 @@
 
 Jericho Opsima
 
+Jenny Cano
+Kzel Lecomes
+
```
Comment (Cano-J) [inline README.md:15]: hello kuya jenny :>

Comment (Rexphillip771) [reply README.md:15]: HELLO ATE BABE

Comment (Cano-J) [reply README.md:15]: freeca lecomes

Comment (Cano-J) [reply README.md:15]: bibingka ni shawn

**Pull request merged**